src/miner.erl

```diff
@@ -367,6 +369,31 @@ handle_call({create_block, Metadata, Txns, HBBFTRound}, _From, State) ->
                     end,
                     [],
                     Metadata),
+    %% TODO what do we do about snapshot disagreements?
+    %% IIRC we had agreed to omit
+    SnapshotHash=
+        case blockchain:config(?snapshot_interval, blockchain:ledger(Chain)) of
+            {ok, Interval} ->
+                case (NewHeight - 1) rem Interval == 0 of
+                    true ->
+                        lager:info("XXX trying to add hash ~p", [NewHeight]),
+                        [SH] = lists:foldl(
```

Comment:
badmatching here would presumably halt block production?

Reply:
yes, I need to change this to 2f+1 agreement, and omission if it does not meet that standard.